src/third/redux-persist/createPersistor.js

```diff
@@ -113,7 +113,7 @@ export default function createPersistor (store, config) {
 
 function warnIfSetError (key) {
   return function setError (err) {
-    if (err) { logging.warn('Error storing data for key:', key, err) }
+    if (err) { logging.warn(err, { message: 'Error storing data for key:', key }) }
```

Comment:
Looks like this is the line that generated the Sentry event you quoted.

Ah, so I guess what happened is that the second argument was a string -- and our types say it should really be one of these objects:
```js
type Extras = {| +[key: string]: JSONable |};
```
and it ultimately gets passed to a `setExtras` method. And I guess that really wants the value to indeed be an object, not a string, and ends up enumerating its properties, perhaps like this:
```
> Object.entries("abc")
[ [ '0', 'a' ], [ '1', 'b' ], [ '2', 'c' ] ]
```

Yeah, it would indeed be good to have type-checking in this file.